src/services/review/ReviewRequestService.ts

```diff
@@ -234,6 +234,24 @@ export default class ReviewRequestService {
           },
         }))
 
+        // It is a new comment to the user if any of the following
+        // conditions satisfy:
+        // 1. The review request views table does not contain a record
+        //    for the user and the review request.
+        // 2. The review request views table contains a record for that
+        //    user and review request, but the lastViewedAt entry is NULL.
+        // 3. The review request views table contains a record in the
+        //    lastViewedAt entry, and the comment has a timestamp greater
+        //    than the one stored in the database.
+        const newComments = await this.getComments(
+          sessionData,
+          site,
+          pullRequestNumber
+        )
+        const countNewComments = await Promise.all(
```

Comment:
could we rename the variables to `allComments` and `newComments` - the current `newComments` just seems to be retrieving all comments of a PR

Reply:
Oops, I forgot to adjust this name, done in db3dce3.